openstudiocore/src/qtwinmigrate/qwinwidget.cpp

```diff
@@ -56,6 +56,10 @@
 
 #include <qevent.h>
 
+// DLM: not sure how to do this right, think I only need this for static builds
+#include <QtPlugin>
+Q_IMPORT_PLUGIN(qwindows)
```

Comment:
my understanding was that the only difference between static plugin vs dynamic in qt 5.6 was that with static you needed to link the .lib plugin like you are doing here for qwindows.lib. I don't think there are more or less requirements for what you do in the code.

qsqlite seems to work without special attention.